doc/source/upgrading_guide.rst

```diff
@@ -115,37 +118,39 @@ Having done the full backup (and dump if possible), stop the server:
    instead of :program:`service`. Currently both are supported.
 
 
-.. admonition:: Output of :bash:`rpm -qa | grep Percona-Server`
+.. admonition:: Output of :bash:`rpm -qa | grep percona-server`
 
```

Comment:
In this case it should stay uppercase, otherwise grep won't find anything (PS 5.7 packages are uppercase)